.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+name: release
+on:
+  workflow_run:
+    workflows: ["unit-tests", "linting"]
+    types:
+      - completed
```

Comment:
@kibertoad the only issue I see is that this `completed` state isn't good enough. Meaning, it will evaluate to true even if the workflows complete in a "failed" situation where linting or unit tests don't actually pass.

Did a quick search, and apparently it's not as straight-forward to just base it on `completed` because it means the workflow finished, regardless of success or failure  (see here for source of conversation: https://github.community/t/workflow-run-completed-event-triggered-by-failed-workflow/128001)

I'm seeing what kind of workarounds we can have, besides the obvious of putting all the work into one workflow file and having conditions inside it.

Reply:
I committed changes for some conditionals, let's see if they work as I merge it in now.